### ISSUE TITLE
Input element does not get focus when clicking the edit button

### ISSUE THREAD

**Issue by jcoyne** [LD4P/sinopia_editor#2486]
**Describe the bug**

When you click the edit button the literal field should have focus.  The code looks like it's giving focus, but it's not actually working.

<img width="292" alt="Screen Shot 2020-09-16 at 12 31 52 PM" src="https://user-images.githubusercontent.com/92044/93371915-b81c7e00-f818-11ea-9c9f-b18fc0b09fa9.png">

https://github.com/LD4P/sinopia_editor/blob/master/src/components/editor/property/InputLiteral.jsx#L55

**Comment by justinlittman** [comment]
There is no longer an edit button.
